controller/scan/gitlab.go

```diff
@@ -162,7 +162,7 @@ func (r *gitlab) getProjects() ([]gitProject, error) {
 	ur := r.gitUrl("/projects")
 	smd.scanLog.WithFields(log.Fields{"url": ur}).Debug("")
 	if data, err := r.getData(ur); err == nil {
-		err = json.Unmarshal(data, &all)
+		_ = json.Unmarshal(data, &all)
```

Comment:
should this line be
`e1 = json.Unmarshal(data, &all)`
?

Reply:
yes, you're right.  I already changed it.